src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -26,6 +26,26 @@ export interface ConnectorInvoice {
   type: "received";
 }
 
+// @Todo: extend missing properties accordingly
+export interface ConnectorTransaction {
+  custom_records: Record<number, number>;
+  fee: number;
+  keysend: boolean;
+  memo: string;
+  payment_hash: {
```

Comment:
I think payment_hash and r_hash should just be a string. Can they be converted in the lndhub connector?

Reply:
maybe only the payment_hash is needed (I think they are both the same thing)

Reply:
So far they are not mapped, because they are not used in the mapped Transaction interface.

https://github.com/lujakob/lightning-browser-extension/blob/4fb06da930b721095a86120643ced30c423aab76/src/extension/background-script/actions/payments/listByAccount.ts#L44-L53